File: dotnet/System/Workspace/Adapters/Allors.Workspace.Adapters.Json/Database/Push/PushResult.cs
// <copyright file="RemotePushResult.cs" company="Allors bv">
// Copyright (c) Allors bv. All rights reserved.
// Licensed under the LGPL license. See LICENSE file in the project root for full license information.
// </copyright>

namespace Allors.Workspace.Adapters.Json
{
    using Allors.Protocol.Json.Api;

    public class PushResult : Result, IPushResult
    {
        public PushResult(IWorkspace workspace, Response response) : base(workspace, response)
        {
        }
    }
}

File: typescript/workspace/system/adapters-json/src/lib/database/push/push-result.ts
import { PushResponse } from '@allors/database-system-protocol-json';
import { IPushResult, ISession } from '@allors/workspace-system-domain';
import { Result } from '../result';

export class PushResult extends Result implements IPushResult {
  constructor(session: ISession, response: PushResponse) {
    super(session, response);
  }
}
